Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -32,8 +32,14 @@ MemoryViewWidget::MemoryViewWidget(QWidget* parent) : QTableWidget(parent)
   verticalHeader()->hide();
   setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
   setShowGrid(false);
+  setAlternatingRowColors(true);
 
   setFont(Settings::Instance().GetDebugFont());
+  QFontMetrics fm(Settings::Instance().GetDebugFont());
+
+  // Row height as function of text size. Less height than default.
+  const int fonth = fm.height() + 3;
+  verticalHeader()->setMaximumSectionSize(fonth);
```

Comment:
Why is this `fm.height() + 3` when #7697 only uses `fm.height() + 1` for `verticalHeader()->setMaximumSectionSize(...)`?

Reply:
Memory View being a series of numbers is too cramped and hard to decipher with a very tight row height. Code view looks okay to me.